plotting.py

```diff
@@ -114,6 +116,70 @@ def plot_inputs(data, particles_dict):
 			particles_dict
 			)
 
+def plot_NN(yhat, data):
+	'''
+    Args:
+        yhat: an ndarray of the probability of each event for each class
+        data: dictionary containing relevant data
+     Returns:
+     	a plot of the probability that each event in a known classes is predicted to be in a specific class
+    '''
+	y_test=data['y_test']
+	w_test=data['w_test']
+	matplotlib.rcParams.update({'font.size': 16})
+	fig = plt.figure(figsize=(11.69, 8.27), dpi=100)
+	bins = np.linspace(0,1,30)
+	#find probability of each class 
+	for k in range(len(np.unique(y_test))):
+		print k
+		color = iter(cm.rainbow(np.linspace(0, 1, len(np.unique(y_test)))))
+		#find the truth label for each class
+		for j in range (len(np.unique(y_test))):
+			c = next(color)
+			_ = plt.hist(yhat[:,k][y_test==j], 
+			bins=bins, 
+			histtype='step', 
+			normed=True, 
+			label='Y=' + str(j),
+			weights=w_test[y_test == j],
+			color=c, 
+			linewidth=1)
+		plt.xlabel('Probabilty of Y=' +str(k)) 
+		plt.ylabel('Weighted Normalized Number of Events')
+		plt.legend()
+		plt.savefig('/Users/gigifstark/CERN_Work/HH2YYBB')
+
+def plot_roc_Curve(yhat, data, le, model_name):

```

Comment:
be consistent with naming practices (no capital letters); also I would suggest a slightly more accurate name, like `save_roc_curves`